modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -15,19 +15,22 @@ Verify your image set configuration by performing a dry run without mirroring an
 +
 [source,terminal]
 ----
-$ oc mirror -c <image_set_config_yaml> --from file://<oc_mirror_workspace_path> docker://<mirror_registry_url> --dry-run --v2
+$ oc mirror -c <image_set_config_yaml> file://<oc_mirror_workspace_path> --dry-run --v2
 ----
++
 Where:
+
 - `<image_set_config_yaml>`: Use the image set configuration file that you just created.
```

Comment:
Remove the hyphens from these items since they don't need to be a bullet list. Also, add another colon after the replaced value:
```suggestion
`<image_set_config_yaml>`:: Use the image set configuration file that you just created.
```